src/bundler/bundler.rs

```diff
@@ -9,3 +41,39 @@ impl Bundler {
         Self { wallet }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::BundlerOpts;
+    use clap::Parser;
+    use ethereum_types::{Address, U256};
+    use std::str::FromStr;
+
+    #[test]
+    fn test_bundle_opt() {
```

Comment:
This is my personal preference, but I do not write prefix `test_`before test names. What do you think? We should have the same naming across codebase (decide for the one).

Reply:
> We should have the same naming across codebase (decide for the one).

Agree.

So you prefer just `bundle_opt`?

Reply:
Yes.